backend/src/test/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersControllerIT.java

```diff
@@ -106,4 +108,53 @@ void testGetUserById_Forbidden() throws Exception {
 			.andExpect(status().isForbidden());
 	}
 
+	@Test
+	@DisplayName("Test authenticated POST /api/v1/users/{userid}/email-validations with valid code")
+	@WithMockUser(roles = { "Users.Administer" })
+	void testVerifyConfirmationCodeStatus_Valid() throws Exception {
+		final var mockUser = ImmutableUser.builder()
+			.id("00000000-0000-0000-0000-000000000000")
+			.email("user@example.com")
+			.emailVerified(false)
+			.addUserAttributes(ImmutableUserAttribute.builder()
+				.name("EXAMPLE_ATTRIBUTE")
+				.value("42")
+				.build())
+			.addConfirmationCodes(ImmutableConfirmationCode.builder().code("code value").build())
+			.build();
+
+		when(userService.getUserById(any())).thenReturn(Optional.of(mockUser));
+		when(userService.verifyEmail(anyString(), anyString())).thenReturn(true);
+
+		mockMvc.perform(post("/api/v1/users/00000000-0000-0000-0000-000000000000/email-validations")
+				.with(csrf()).header("origin", "http://localhost")
+				.content("{\"code\": \"code value\"}"))
```

Comment:
This doesn't match what the actual endpoint expects, but because the validation is broken in the controller, this test is actually a false positive.

Reply:
Both this test and endpoint in the controller have been updated as of commit 70d88ff21299e8b6f77785dd0b22d60b10f1a733.